packages/client/src/forms/register/fieldMappings/birth/query/registration-mappings.ts

```diff
@@ -306,6 +309,24 @@ export const changeHirerchyQueryTransformer =
     return transformedData
   }
 
+const getUserFullName = (history: History): string => {
+  return history?.user ? getUserName(history.user) : ''
+}
+
+const getUserRole = (history: History): MessageDescriptor => {
+  return (
+    (history?.user && userMessages[history.user.systemRole]) || {
+      defaultMessage: ' ',
+      description: 'empty string',
+      id: 'form.field.label.empty'
+    }
+  )
+}
```

Comment:
This will return systemRole. We probably want the `role` to be returned from here instead